devel/smake/Portfile

```diff
@@ -1,41 +1,96 @@
-PortSystem 1.0
-PortGroup  compiler_blacklist_versions 1.0
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-name			smake
-version			1.2.5
-categories		devel
-platforms		darwin
-maintainers		nomaintainer
-license			CDDL
-description		highly portable make program with automake features
-long_description	Smake is a highly portable make program with automake \
-				features. It is currently mainly targeted to be used \
-				with the makefiles system. 
+PortSystem              1.0
+PortGroup               makefile 1.0
+PortGroup               compiler_blacklist_versions 1.0
+PortGroup               conflicts_build 1.0
+PortGroup               codeberg 1.0
 
-homepage		https://sourceforge.net/projects/s-make/
-master_sites	sourceforge:project/s-make
-use_bzip2		yes
+codeberg.setup          schilytools schilytools 2023-04-19
 
-checksums           rmd160  f683a4051273b3e24e45a2b435ddb23509db55bf \
-                    sha256  27566aa731a400c791cd95361cc755288b44ff659fa879933d4ea35d052259d4
+name                    smake
+version                 1.7-${codeberg.version}
+revision                0
+categories              devel
+platforms               darwin
```

Comment:
`platforms darwin` is the default, so that line can be removed.

Reply:
Will do.  It was in the original Portfile so I kept it.  But it is easy to remove.